cronenbroguelike/commands.py

```diff
@@ -163,9 +160,9 @@ def _resolve_attack(attacker, attack):
     defender = attack.target
     is_player = attacker is G.player
     if is_player:
-        subj, obj = ["you", defender.name]
```

Comment:
obj is now used below; please remove this from your change. Otherwise good to go :)